CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added
+
+- Add manually run task to cleanup unused integrations @mderynck ([#3941](https://github.com/grafana/oncall/pull/3941))
```

Comment:
Not sure if that's have to be a manual task. I'd prefer it to b automatic, otherwise it is not clear who will own it, and we would need to add some docs about that process

Reply:
I would like to release this task to run manually and then in a follow-up PR connect it to run as part of sync_organization to cover everything. So it will only be manual for a short time.